argus/backends/tempest/tempest_backend.py

```diff
@@ -77,11 +77,14 @@ def __init__(self, conf, name, userdata, metadata, availability_zone):
 
     def _configure_networking(self):
         subnet_id = self._manager.primary_credentials().subnet["id"]
-        self._manager.network_client.update_subnet(
+        self._manager.subnets_client.update_subnet(
             subnet_id,
             dns_nameservers=self._conf.argus.dns_nameservers)
 
     def _create_server(self, wait_until='ACTIVE', **kwargs):
+        for key, value in list(kwargs.items()):
```

Comment:
What is this code meant for? Also I would appreciate if you don't merge patches if I don't review them. @ionuthulub